packages/next/src/server/app-render.tsx

```diff
@@ -236,7 +238,19 @@ function createErrorHandler(
     }
     // Used for debugging error source
     // console.error(_source, err)
-    console.error(err)
+
+    // Don't log the suppressed error during export
+    if (
+      !(
+        isNextExport &&
+        err.message.includes(
```

Comment:
Do we validate `err.message` is a valid string further up already?

Reply:
Good catch, thanks!